M/MAGMA/build_tarballs.jl

```diff
@@ -30,9 +43,17 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    BuildDependency(PackageSpec(name="CUDA_full_jll", version=v"11.6.0")),
-    Dependency("libblastrampoline_jll")
+    # You can only specify one cuda version in the deps. To build against more than 
+    # one cuda version, you have to include them as Archive Sources. (see Torch_jll)
+    RuntimeDependency(PackageSpec(name="CUDA_Runtime_jll")),
+    BuildDependency(PackageSpec(name="CUDA_full_jll", version=v"11.0.3"), platforms=platforms),
+    Dependency("libblastrampoline_jll"),
```

Comment:
It's better to specify compat with libblastrampoline, something like 5.2.0 or so